docs/faq.md

```diff
@@ -30,6 +30,9 @@ After all relevant objects gets deleted from Avi, the reason is changed to objDe
 
 To re-create the objects in Avi, the configmap has to be edited to set deleteConfig: "false". After this, the condition in ako statefulset of type ako.vmware.com/ObjectDeletionInProgress is deleted automatically.
 
+**Note**: To cleanup the shared VS VIP that is retained, the user has to:
```

Comment:
should it be part of section "#### How is the Shared VS lifecycle controlled" with more elaboration of shared vs vip behaviour change?

Reply:
I have moved it to the section `How is the Shared VS lifecycle controlled`.